cards/abg-interpretation/card.md

```diff
@@ -5,6 +5,7 @@ authors:
     - 'Michelle Lin, MD'
 created: 2010/04/02
 updates: null
++updates: 2017/02/24
```

Comment:
This needs to be fixed. Two issues:
1. The obvious one: Remove the duplicate and delete the stray `+`
2. The `updates` field` is an array of strings, not a string.

Also, I believe we are keeping categories lowercase.

It should look like this:

```yml
title: 'ABG Interpretation'
authors:
    - 'Michelle Lin, MD'
created: 2010/04/02
updates:
    - 2017/02/24
categories:
    - pulmonology
drugs: null
```